util/util.go

```diff
@@ -109,6 +109,9 @@ func GetProviders() []provider.Provider {
 			if key == "discord" && c == "webhook" && len(strings.TrimSpace(v.(string))) > 0 {
 				providers = append(providers, provider.NewDiscord(viper.GetString("alert.discord.webhook")))
 			}
+			if key == "telegram" && c == "token" && len(strings.TrimSpace(v.(string))) > 0 {
+				providers = append(providers, provider.NewTelegram(viper.GetString("alert.telegram.token"), viper.GetString("alert.telegram.chatId")))
+			}
```

Comment:
@Andrew44Ashraf do we need to add a check for the chat id as well?

Reply:
@abahmed updated